docs/marketplace/dev/feature-walkthroughs/202108.0/merchant-opening-hours-feature-walkthrough.md

```diff
@@ -13,6 +13,23 @@ To learn more about the feature and to find out how end users use it, see [Merch
 
 {% endinfo_block %}
 
+## Module dependency graph
+
+![Module Dependency Graph](https://confluence-connect.gliffy.net/embed/image/0b05a957-57a4-4422-9595-5bbe63a6a18b.png?utm_medium=live&utm_source=custom)
```

Comment:
- please mage Merchant module Gray since it doesn't belong to this feature
- please remove DataImport and Storage modules
- MerchantOpeningHoursStorage also depends on Merchant, please recheck other dependencies